runtime/src/accounts_db.rs

```diff
@@ -9950,6 +10000,31 @@ pub mod tests {
         }
     }
 
+    #[test]
+    fn test_combine_multiple_slots_into_one_at_startup() {
+        solana_logger::setup();
+        let (db, slot1) = create_db_with_storages_and_index(false, 2, None);
+        let slot2 = slot1 + 1;
+
+        let initial_accounts = get_all_accounts(&db, slot1..(slot2 + 1));
```

Comment:
*minor*

There is a syntax for ranges that include the upper bound.  You may use it here:
```suggestion
        let initial_accounts = get_all_accounts(&db, slot1..=slot2);
```

Reply:
the fn takes `Range<Slot>`. It is a test only function. Making it inclusive upper bounds changes the type. Generic ranges are a little of their own hassle, in my experience. So that is why it is exclusive bounds.

Reply:
Here is a simple fix: https://github.com/solana-labs/solana/pull/30581
`get_all_accounts` does not really care if the argument is a range or not.
It wants a source of `slot` indices.  And can be happy with an iterator.
All ranges are also iterators.